installlib/installcommand.class.php

```diff
@@ -96,7 +100,11 @@ protected function execute(InputInterface $input, OutputInterface $output) {
 		//STATIC???
 		define("AMP_CONF", "/etc/amportal.conf");
 		define("ODBC_INI", "/etc/odbc.ini");
-		define("ASTERISK_CONF", "/etc/asterisk/asterisk.conf");
+		if (PHP_OS == "FreeBSD") {
+			define("ASTERISK_CONF", "/usr/local/etc/asterisk/asterisk.conf");
```

Comment:
While I am ok with this it should probably be a CLI parameter when it comes down to it